catalogs/catalog-hive/build.gradle.kts

```diff
@@ -77,12 +77,12 @@ dependencies {
 tasks {
   val copyDepends by registering(Copy::class) {
     from(configurations.runtimeClasspath)
-    into("build/libs")
+    into("build/libs_all")
```

Comment:
What is the purpose of this change?

Reply:
The `build` and `copyDepends` tasks of `catalog-hive` use the same output directory 'build/libs', which is needed by task `shadowJar` in `bundled-catalog`, gradle will get errors and says "task `shadowJar` depends on task `copyDepends`".

Reply:
You'd better add comment on the file to describe it.

Reply:
Added.